elastic/kibana
# [Synthetics] allow saving params for lightweight monitors

## Summary

Relates to https://github.com/elastic/synthetics/issues/725

Allows the `params` value to be saved against lightweight monitors.

This ensures that the formatters for both private and public location can access these params, and apply them as overwrites to global params.

### Testing

1. Check out associated synthetics agent PR 
2. Create a global parameter, for example `GLOBAL_URL`
3. Create lightweight monitor that uses that global parameter in your project file
4. Push the monitor and navigate to Synthetics. Ensure the monitor uses the value of the `GLOBAL_URL` parameter appropriately
5. Navigate back to your project config. In your `synthetics.config.ts` file, add a param for `GLOBAL_URL`
6. Repush your monitor
7. Confirm `GLOBAL_URL` is overwritten by the pushed params
